src/qforte/ucc/spqe.py

```diff
@@ -60,6 +60,30 @@ def run(self,
             use_cumulative_thresh=True,
             max_excit_rank = None,
             optimizer = 'Jacobi'):
+        """
+        spqe_thresh: float
+            The convergence threshold against which the norm of the residual is compared
+            (in L2 norm) to determine convergence.
+        spqe_maxiter: int
+            The maximum number of macroiterations
+        dt: float
+            The numerical. Use as small a value as possible to get numerically stable results.
+        M_omega: Union['inf', float]
+            The number of measurements to use when constructing the estimated residual, eq. 15
```

Comment:
Not sure if we should point to eq. 15 (the residual state) or to 16 (the residual element estimation).